config/src/config/test_data/validator.yaml

```diff
@@ -55,3 +55,8 @@ validator_network:
         ca_certificate: "/full/path/to/certificate"
         token:
             from_disk: "/full/path/to/token"
+
+json_rpc:
+    # Enable streaming RPC (i.e JSON-RPC over websockets)
+    stream_rpc:
+        enabled: true
```

Comment:
I thought the intent was to make this off by default? why have we turned the flag on in all our test configs?

Reply:
because it was simpler than the alternative- I updated the method to allow for specifying websocket